src/dso_api/dynamic_api/middleware.py

```diff
@@ -98,6 +98,16 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             method=request.method,
             request_headers=repr(request.META),
             subject=subject,
-            data=data)
+            data=data,
+        )
 
         audit_log.info(json.dumps(log))
+
+
+class QueryParamLegalityCheckMiddleware(BaseMiddleware):
+    def process_view(self, request, *args, **kwargs):
+        get = request.GET.copy()
+        for query_type, param in request.GET.items():
+            if not param:
+                get.pop(query_type)
```

Comment:
Wat gebeurd er als de invalid query_type wordt verwijderd? Is er een error zichtbaar of gebeurd er niets?

Reply:
Ik zou een 400 kunnen teruggeven in plaats van weggooien. Dat is idd wat schoner